source/includes/steps-install-shell-linux-rpm.yaml

```diff
@@ -0,0 +1,21 @@
+---
+source:
+  file: steps-install-shell-base.yaml
+  ref: navigate-dlc
+ref: navigate-dlc-linux-rpm
+---
+title: "Download the Red Hat 64-bit ``.rpm`` package."
+ref: download-archive-rhel
+level: 4 
+---
+title: "Extract and install the ``.rpm`` package."
+ref: extract-archive-rpm
+level: 4
+content: |
+  To install the |mdb-shell|, run the following command from the
+  directory containing the ``mongosh`` ``.rpm`` package:
+
+  .. code-block:: sh
+
+     sudo yum install -y mongosh-{+version+}-x86_64.rpm
```

Comment:
Does this command work?

I don't have a RHEL-like box to try it but I _think_ this the command to install a package that is available in a repository. Does `yum` also allow installing RPMs directly? I thought one needs to `rpm -i mongosh-{+version+}-x86_64.rpm` but I might very well be wrong.

Reply:
It's [an alias](https://serverfault.com/questions/940791/whats-the-difference-between-yum-install-local-path-and-yum-localinstall) to `localinstall` if passed an RPM package present in `cwd`.

`[vagrant@centos8 tmp]$ wget https://downloads.mongodb.com/compass/mongosh-0.4.0-x86_64.rpm
2020-09-29 14:42:48 (32.7 MB/s) - ‘mongosh-0.4.0-x86_64.rpm’ saved [33599000/33599000]
[vagrant@centos8 tmp]$ sudo yum install -y mongosh-0.4.0-x86_64.rpm
Total size: 32 M
Installed size: 136 M
Downloading Packages:
Running transaction check
Transaction check succeeded.
Running transaction test
Transaction test succeeded.
Running transaction
  Preparing        :                                                                                                                           1/1
  Installing       : mongosh-0.4.0-1.el8.x86_64                                                                                                1/1
  Running scriptlet: mongosh-0.4.0-1.el8.x86_64                                                                                                1/1
  Verifying        : mongosh-0.4.0-1.el8.x86_64                                                                                                1/1

Installed:
  mongosh-0.4.0-1.el8.x86_64

Complete!`

Reply:
Ah, cool. Always happy to learn new stuff, thank you!